packages/media/src/webrtc/stats/IssueMonitor/issueDetectors.ts

```diff
@@ -0,0 +1,271 @@
+import { StatsClient } from "../types";
+
+interface IssueDetector {
+    id: string;
+    global?: boolean;
+    enabled: (args: IssueCheckData) => boolean;
+    check: (args: IssueCheckData) => boolean;
+}
+
+export interface IssueCheckData {
+    client: StatsClient;
+    clients: StatsClient[];
+    kind: string;
+    track: MediaStreamTrack | undefined;
+    trackStats: any;
+    stats: any;
+    hasLiveTrack: boolean;
+    ssrc0: any;
+    ssrcs: any;
+    issues: any;
+    metrics: any;
+}
+
+export const badNetworkIssueDetector: IssueDetector = {
+    id: "bad-network",
+    enabled: ({ hasLiveTrack, ssrcs }) => hasLiveTrack && ssrcs.length > 0,
+    check: ({ client, clients, kind, ssrcs }) => {
+        let hasPositiveBitrate = false;
+
+        ssrcs.forEach((ssrc: any) => {
```

Comment:
suggestion:
const hasPositiveBitrate = ssrcs.some(ssrc => ssrc.bitrate > 0)
(shorter, const, functional and early return)